zuul.d/nodesets.yaml

```diff
@@ -467,6 +467,81 @@
         nodes:
           - ubuntu-bionic
 
+- nodeset:
+    name: vqfx-18.4R1-python27
+    nodes:
+      - name: ubuntu-bionic
+        label: ubuntu-bionic-1vcpu
+      - name: vqfx-18.4R1.8-RE
+        label: vqfx-18.4R1.8-RE
```

Comment:
We are getting a head of ourselves here. We first need to add the node info into nodepool, https://github.com/ansible-network/windmill-config/